docs/SPIRVRepresentationInLLVM.rst

```diff
@@ -239,3 +300,55 @@ For example:
   !9 = !{!7, i32 32}     ; independent forward progress is required for 'kernel2'
   !10 = !{i16 6, i16 123} ; 6 - Generator Id, 123 - Generator Version 
 
+Additional requirements for LLVM module
+=======================================
+
+Target triple and datalayout string
+-----------------------------------
+
+Target triple architecture must be ``spir`` (32-bit architecture) or ``spir64``
+(64-bit architecture) and ``datalayout`` string must be aligned with OpenCL
+environment specification requirements for data type sizes and alignments (e.g.
+3-element vector must have 4-element vector alignment). For example:
+
+.. code-block:: llvm
```

Comment:
This example isn't rendered.

Reply:
Fixed with https://github.com/KhronosGroup/SPIRV-LLVM-Translator/commit/16df0913330b960bcb118648c63c06e7579bb400.